src/wizard/javascript/profiling-onboarding/react/1.install.md

```diff
@@ -0,0 +1,22 @@
+---
+name: React (JS Self Profiling)
+doc_link: https://docs.sentry.io/platforms/javascript/guides/react/profiling/
+support_level: beta
+type: framework
+---
+
+#### Install
+
+Our browser profiling integration is built upon the profiler exposed by the [JS Self-Profiling API](https://wicg.github.io/js-self-profiling/), it is therefor in beta stage and is likely to only to move out as the official spec progresses. There are obviously risks with using a beta package - we've made sure that the SDK gracefully degrades in unsupported environments.
+
+Please note that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherintly be biased towards that demographic - this is something you need to be aware of when making decisions.
```

Comment:
```suggestion
Please note, that since profiling API is currently only implemented in Chromium based browsers, the profiles collected will inherently be biased towards that demographic.
```